specs/chugsplash-manager.md

```diff
@@ -123,8 +131,9 @@ const getProxyByName = (
 * Cancelling a bundle MUST put the bundle in the `CANCELLED` state.
 * Cancelling a bundle MUST remove the active bundle.
 * If an executor has been selected within the last `executionLockTime` seconds (i.e. `block.timestamp` <= `timeClaimed + executionLockTime`):
-  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `BUNDLE_EXECUTION_BOND_AMOUNT`.
+  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `BUNDLE_EXECUTION_BOND_AMOUNT + executorBondAmount`.
```

Comment:
Unrelated to the new action type.

We must add the `executorBondAmount` to the `debt` here so that the executor can withdraw the full amount owed to them.